VultisigApp/VultisigApp/Views/Vault/ChainDetailView.swift

```diff
@@ -132,14 +132,22 @@ struct ChainDetailView: View {
         ChainHeaderCell(group: group, isLoading: $isLoading)
     }
     
+    var sortedCoins: [Coin] {
```

Comment:
As discussed yesterday , this doesn't help with the performance , can you move it into viewmodel as a published property instead , only update it when the balance get changed

Reply:
I will remove that.